velox/functions/sparksql/Bitwise.cpp

```diff
@@ -1,160 +0,0 @@
-/*
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-#include "velox/functions/sparksql/Bitwise.h"
-
-namespace facebook::velox::functions::sparksql {
-
-template <typename T>
-struct BitwiseAndFunction {
```

Comment:
Why do we need this change to move the implementations from cpp file to the header?

Reply:
keep same format with others.

Reply:
after move registration code from Bitwise.h/cpp, no need to keep cpp file any more, just like Arithmetic.h